test/unit/integrations/payu_in/payu_in_notification_test.rb

```diff
@@ -30,10 +30,6 @@ def test_accessors
     assert_equal true, @payu.checksum_ok?
   end
 
-  def test_compositions
-    assert_equal '10.00', @payu.gross
```

Comment:
The composition tests basically asserted that gross/gross_cents would build the expected money object. This removed test case is a duplicate of the a `test_accessors` case, defined on line 17.